docs/theme/components/collocator/index.tsx

```
@@ -0,0 +1,136 @@
+
+  const renderHeader = (children: String | React.ReactElement, className?: string) => (
+    <div className={classNames(styles.header, className)}>{children}</div>
+  );
+
+  const renderFunctions = () => (
+    <div className={styles.functions}>
+      <Codesandbox code={examples.code} className={classNames(styles.icon, styles.codesandBox)} tip='更多组合用法, 可前往Playground...' />
+      {functions.map((item, index) => (
+        <Tooltip tip={item.tip} trigger='hover' position='top'>
+          <div
+            key={index}
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)